backend/src/main/java/gov/cdc/usds/simplereport/service/ScheduledTasksService.java

```diff
@@ -0,0 +1,32 @@
+package gov.cdc.usds.simplereport.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@Service
+public class ScheduledTasksService {
+    private static final Logger LOG = LoggerFactory.getLogger(ScheduledTasksService.class);
+    private static final int FIXED_DELAY_MS = 1000*60*5;  // fixed period in milliseconds after last task ends
+
+    @Scheduled(fixedDelay = FIXED_DELAY_MS)
+    public void reportCurrentTime() {
+        LOG.info("Fixed puts a delay between tasks, so they can run as long as needed without overlapping.\n" +
```

Comment:
The default log format does actually include the time. 🙂

Reply:
Ha! You're absolutely right.